### ISSUE TITLE
[1.0.14] Samsung Galaxy Tab S 10.5 / Android 4.2.2 (Asus Nexus 7  / Android 5.1.1) - Create - Time to load motifs is very long

### ISSUE THREAD

**Issue by vysohanych** [Lomotif/lomotif-android-issues#43]
Frequency: Some Time

Action:
1. Launch Lomotif
2. Tap START button on the home screen
3. Select a few motifs on the Album screen
4. Tap apply button on the Album screen
5. Select any song on Find Music screen
6. Tap Use 30 Seconds Preview button

Actual result: Time to load motifs is very long

**Comment by xiuie** [comment]
@vysohanych How many Motifs did you select? And how long does it take? Approximately how many times longer than iOS version?

**Comment by hiteshbheda** [comment]
@xiuie I can replicate this issue in Asus Zenfone Go 4.5 / Android 5.1 and also in Sony xPeria Z / Android 5.1.1.

Approximately it takes 3 times more time compared to iOS version.

**Comment by vysohanych** [comment]
@xiuie I agree with @hiteshbheda . I use about 3-5 lomotifs and they are loading more 3 times than iOS
